socket_mode.go

```diff
@@ -0,0 +1,35 @@
+package slack
+
+import (
+	"context"
+)
+
+// SocketModeConnection contains various details about the SocketMode connection.
+// It is returned by an "apps.connections.open" API call.
+type SocketModeConnection struct {
+	URL  string                 `json:"url,omitempty"`
+	Data map[string]interface{} `json:"-"`
+}
+
+type openResponseFull struct {
+	SlackResponse
+	SocketModeConnection
+}
+
+// StartSocketModeContext calls the "apps.connections.open" endpoint and returns the provided URL and the full Info block with a custom context.
+//
+// To have a fully managed Websocket connection, use `NewRTM`, and call `Run()` on it.
```

Comment:
```suggestion
// To have a fully managed Websocket connection, use `New()`, and call `Run()` on it.
```

Reply:
Fixed in fcf2b1e. I've also added mention to `slackmode.Client` and edited the message to refer to Socket Mode